Userland/Libraries/LibJS/JIT/Compiler.cpp

```diff
@@ -1088,43 +1119,195 @@ void Compiler::compile_mul(Bytecode::Op::Mul const& op)
         static Value cxx_##snake_case_name(VM& vm, Value lhs, Value rhs)                 \
         {                                                                                \
             return TRY_OR_SET_EXCEPTION(snake_case_name(vm, lhs, rhs));                  \
-        }                                                                                \
-                                                                                         \
-        void Compiler::compile_##snake_case_name(Bytecode::Op::TitleCaseName const& op)  \
```

Comment:
Why not extend `JS_ENUMERATE_COMPARISON_OPS` with a floating point condition to keep this function in the macro to reduce the code?

Reply:
I don't know if you use a formatter for your C++ code, but when you use `clang-format` working with multi line macro's is quite simple, because the formatter will align all the spaces for the backslash.
I would still argue to put the function inside the macro to reduce code and that only `i32 < i32`, `double < double`, `double < i32`, `double < double`, fast paths are needed because those are the most common ones, other data types are quite rare and could fall back to C++, they won't be worth the JITTED code size.